swatch-contracts/src/test/java/com/redhat/swatch/contract/security/RolesAugmentorTest.java

```diff
@@ -42,15 +42,17 @@ private QuarkusSecurityIdentity securityIdentityForRhIdentityJson(String json) {
     }
   }
 
+  private QuarkusSecurityIdentity securityIdentityForAnonymous() {
+    return QuarkusSecurityIdentity.builder().setAnonymous(true).build();
+  }
+
   @Test
   void testRoleGrantedToRhIdentityWhenEnabled() {
     var augmentor = new RolesAugmentor();
     augmentor.testApisEnabled = true;
-    SecurityIdentity x509Identity =
-        securityIdentityForRhIdentityJson(RhIdentityUtils.X509_IDENTITY_JSON);
     var subscriber =
         augmentor
-            .augment(x509Identity, null)
+            .augment(securityIdentityForAnonymous(), null)
```

Comment:
Maybe a new test should be added for anonymous that verifies the test role is removed?

Reply:
I added one.  `testNoTestRoleGrantedToRhIdentityWhenDisabled`